modules/os2forms_digital_post/os2forms_digital_post.services.yml

```diff
@@ -8,64 +8,35 @@ services:
     arguments: [ 'webform_submission' ]
 
   Drupal\os2forms_digital_post\Helper\Settings:
-    arguments:
-      - "@config.factory"
-      - "@key.repository"
+    autowire: true
 
   Drupal\os2forms_digital_post\Helper\CertificateLocatorHelper:
-    arguments:
-      - "@Drupal\\os2forms_digital_post\\Helper\\Settings"
-      - "@key.repository"
+    autowire: true
 
   Drupal\os2forms_digital_post\Helper\MeMoHelper:
+    autowire: true
     arguments:
-      - "@Drupal\\os2forms_digital_post\\Helper\\Settings"
-      - "@plugin.manager.element_info"
-      - "@webform.token_manager"
+      $elementInfoManager: "@plugin.manager.element_info"
+      $webformTokenManager: "@webform.token_manager"
```

Comment:
Are these really needed? Can we use

```
function __construct(
  …
  #[\Symfony\Component\DependencyInjection\Attribute\Autowire]
  $elementInfoManager,
  …
)
```

instead? Using an attribute may make the code easier to maintain.

Reply:
Makes sense. `phpcs` will deem this a possible useless override, so i have ignore this specific sniff.